fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -553,12 +553,13 @@ public class Config extends ConfigBase {
 
     /**
      * If true, FE will reset bdbje replication group(that is, to remove all electable nodes' info)
-     * and is supposed to start as Leader.
+     * and is supposed to start as Leader. After reset, this node will be the only member in this cluster,
+     * and the others node should be rejoin to this cluster by `Alter system add/drop follower/observer 'xxx'`;
      * If all the electable nodes can not start, we can copy the metadata
      * to another node and set this config to true to try to restart the FE.
      */
     @ConfField
-    public static String metadata_failure_recovery = "false";
+    public static String reset_election_group = "false";
```

Comment:
using `alias` annotation

Reply:
Better not to use the old name, because this config is often used incorrectly by users.

Reply:
prefix with `bdbje`?